modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/extensionflow/MathExtensionTestCase.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.integration.test.extensionflow;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.databridge.commons.Event;
+import org.wso2.carbon.event.simulator.stub.types.EventDto;
+import org.wso2.carbon.integration.test.client.Wso2EventServer;
+import org.wso2.cep.integration.common.utils.CEPIntegrationTest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Testing Math Ceil Extension
+ */
+public class MathExtensionTestCase extends CEPIntegrationTest {
+	private static final Log log = LogFactory.getLog(MathExtensionTestCase.class);
+
+	@BeforeClass(alwaysRun = true) public void init() throws Exception {
+
+		super.init(TestUserMode.SUPER_TENANT_ADMIN);
+		String loggedInSessionCookie = getSessionCookie();
+
+		eventSimulatorAdminServiceClient = configurationUtil
+				.getEventSimulatorAdminServiceClient(backendURL, loggedInSessionCookie);
+		eventStreamManagerAdminServiceClient = configurationUtil
+				.getEventStreamManagerAdminServiceClient(backendURL, loggedInSessionCookie);
+		eventPublisherAdminServiceClient = configurationUtil
+				.getEventPublisherAdminServiceClient(backendURL, loggedInSessionCookie);
+		eventProcessorAdminServiceClient = configurationUtil
+				.getEventProcessorAdminServiceClient(backendURL, loggedInSessionCookie);
+	}
+
+	@Test(groups = {
+			"wso2.cep" }, description = "Testing Math Ceil Siddhi Extension") public void siddhiMathExtensionTestScenario()
+			throws Exception {
+		final int messageCount = 3;
+
+		int startESCount = eventStreamManagerAdminServiceClient.getEventStreamCount();
+		int startEPCount = eventPublisherAdminServiceClient.getActiveEventPublisherCount();
+		int startEXPCount = eventProcessorAdminServiceClient.getExecutionPlanConfigurationCount();
+
+		//Add StreamDefinition
+		String streamDefinitionAsString1 = getJSONArtifactConfiguration("extensionflows/math",
```

Comment:
Thanks. This has been already fixed in commit https://github.com/thiliA/product-cep/commit/678409ddbce18baebacefaa5f4016d758e07246e